docs/build.gradle

```diff
@@ -120,6 +120,8 @@ testClusters.matching { it.name == "yamlRestTest"}.configureEach {
   // TODO: remove this once cname is prepended to transport.publish_address by default in 8.0
   systemProperty 'es.transport.cname_in_publish_address', 'true'
 
+  systemProperty 'es.queryable_built_in_roles_enabled', 'false'
```

Comment:
Only disabling it to avoid adjusting docs test setups. Will enable it in a separate followup PR.

Reply:
Raised https://github.com/elastic/elasticsearch/pull/120740

Reply:
It only impacts a single test. See the linked PR.

Reply:
I see. Thanks for clarifying!